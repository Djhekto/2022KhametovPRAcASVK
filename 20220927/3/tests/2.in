0, 0, 0
4, 3, 2
5, 5, 5
8, 9, 4
4, 3, 2
1, 1, 1