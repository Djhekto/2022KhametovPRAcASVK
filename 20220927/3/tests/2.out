0,0,0
46,47,24
65,65,35